frontend/src/app/components/dropdowns/user-menu/user-menu.component.less

```diff
@@ -8,6 +8,9 @@ div.user-menu {
     ul {
         list-style-type: none;
         width: 100%;
+        li {
+            cursor: pointer;
+        }
```

Comment:
Adds the hand-cursor to the "log out" button. I _could not_ figure out where the `cursor` CSS directive was coming from for the rest of the list though--is there a better place for this?